CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-* [Unreleased](#unreleased)
+* [1.4.6](#146---2023-09-12)
```

Comment:
I'm just guessing on the data, here. I'll change it, as needed.

Reply:
Updated the date to today. The historical significance of the date is unfortunate, but not a good enough reason to make clients wait an extra day for the goods.